spec/class.dd

```diff
@@ -997,15 +997,15 @@ $(GNAME Allocator):
 $(P A class member function of the form:)
 
 ------
-new(uint size)
+new(size_t size)
 {
     ...
 }
 ------
 
         is called a class allocator.
         The class allocator can have any number of parameters, provided
-        the first one is of type uint.
+        the first one is of type size_t.
```

Comment:
try 
```
the first one is of type $(D size_t).
```

Reply:
I guess this is causing an error in the PDF generator.